### ISSUE TITLE
Need Human-readable output for get-nodes

### ISSUE THREAD

**Issue by dopheide-esnet** [zeek/zeek-client#26]
I've been playing around with a human-readable table for get-nodes.  I've got working code as an example, but I feel like we can wait a bit to see what other output commands we may want to support.

https://github.com/dopheide-esnet/zeek-client/commit/858e06c748db7611542835fd3be153ddc0a296b8

**Comment by ckreibich** [comment]
Yeah this is great, thank you for exploring — I fully agree we don't want the JSON output as the default thing to consume by humans. The only question is how to structure/standardize its processing.
